data/src/main/java/com/bottlerocketstudios/brarchitecture/data/converter/LinksDtoConverter.kt

```diff
@@ -0,0 +1,14 @@
+package com.bottlerocketstudios.brarchitecture.data.converter
+
+import com.bottlerocketstudios.brarchitecture.data.model.LinkDto
+import com.bottlerocketstudios.brarchitecture.data.model.LinksDto
+import com.bottlerocketstudios.brarchitecture.domain.models.Link
+import com.bottlerocketstudios.brarchitecture.domain.models.Links
+
+fun LinksDto.convertToLinks(): Links {
+    return Links(avatar = avatar?.convertToLink())
+}
```

Comment:
These could be single expression functions.

Reply:
ooo. Good Idea. I will change them.